handlers/admin/data-contributions.js

```diff
@@ -1,13 +1,16 @@
 'use strict';
 
 const Joi = require('joi');
+const DataCategory = require('../../models/data-category');
 const DataContribution = require('../../models/data-contribution');
 
 
 function editDataContribution(request, reply) {
   const data = {
     approved: request.payload.approved,
-    curation_comments: request.payload.curation_comments,
+    trial_id: request.payload.trial_id || null,
+    data_category_id: request.payload.data_category_id,
+    curation_comments: request.payload.curation_comments || null,
```

Comment:
Yeah, nunjucks is not the most friendly system with undefined and null values. It will attempt to cast them into a literal string quite often. I don't remember exactly what was the issue here, but I'd dare guessing that if it were `undefined` then nunjucks would fail its `{% if curation_comments %}` blocks due to casting the `undefined` into `'undefined'`. Honestly though, I don't remember exactly, want me to check?